<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ihm</class>
 <widget class="QMainWindow" name="Ihm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>778</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QDeclarativeView" name="canvas">
      <property name="minimumSize">
       <size>
        <width>760</width>
        <height>760</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>760</width>
        <height>760</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <widget class="QMenu" name="menuRules">
     <property name="title">
      <string>Rules</string>
     </property>
     <addaction name="actionDouble"/>
     <addaction name="actionEating"/>
    </widget>
    <addaction name="actionNewGame"/>
    <addaction name="menuRules"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTime">
    <property name="title">
     <string>Time</string>
    </property>
    <addaction name="action10ms"/>
    <addaction name="action20ms"/>
    <addaction name="action50ms"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuTime"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDouble">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Double 3</string>
   </property>
  </action>
  <action name="actionEating">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manger!!</string>
   </property>
  </action>
  <action name="actionNewGame">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
  <action name="action10ms">
   <property name="text">
    <string>10ms</string>
   </property>
  </action>
  <action name="action20ms">
   <property name="text">
    <string>20ms</string>
   </property>
  </action>
  <action name="action50ms">
   <property name="text">
    <string>50ms</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
